tests/python_package_test/test_basic.py

```diff
@@ -430,6 +430,52 @@ def test_cegb_scaling_equalities(tmp_path):
         assert p1txt == p2txt
 
 
+def test_cegb_split_buffer_clean():
```

Comment:
Add some asserts. Refer to https://github.com/microsoft/LightGBM/pull/5157#discussion_r868526686.

Reply:
Resolved via a04d004